<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <node pkg="raw_imu_decode" type="raw_imu_reconstructor" name="raw_imu_reconstructor_node" output="screen">
        <param name="raw_imu_filename" value="/home/csl/dataset/data_515/imu1/raw_data.imu" type="string"/>
        <param name="dst_imu_adis"
               value="/home/csl/dataset/data_515/data_202351517465/imu1_adis_new.txt" type="string"/>
        <param name="image_timestamp"
               value="/home/csl/dataset/data_515/data_202351517465/stamp_rearrange_cam_gs.txt"
               type="string"/>
        <param name="gps_week" value="2262" type="int"/>
    </node>
</launch>